pdns/rec-snmp.cc

```diff
@@ -108,6 +108,8 @@ static const oid policyResultNodataOID[] = { RECURSOR_STATS_OID, 89 };
 static const oid policyResultTruncateOID[] = { RECURSOR_STATS_OID, 90 };
 static const oid policyResultCustomOID[] = { RECURSOR_STATS_OID, 91 };
 static const oid queryPipeFullDropsOID[] = { RECURSOR_STATS_OID, 92 };
+static const oid dnssecAuthenticDataQueriesOID[] = { RECURSOR_STATS_OID, 95 };
```

Comment:
I was a bit surprised a first by the gap between 92 and 95, but I guess it might make sense to keep the same OIDs between 4.2.x and 4.1.x.